doc/glm/index.rst

```diff
@@ -7,11 +7,18 @@
 Analyzing fMRI using GLMs
 =========================
 
-Nilearn's stats module uses Generalized Linear Model to
-analyse functional MRI data.
+Nilearn's stats module uses Generalized Linear Model to analyse functional MRI data. Nilearn
+offers functions to set up first- and second-level models, perform t-tests, multiple-comparisons
+correction and thresholding, and allows for the generation of html reports that detail the
+analyses performed and present thresholded results.
+
+
 
 .. include:: ../includes/big_toc_css.rst
 
+
 .. toctree::
 
    glm_intro.rst
```

Comment:
Maybe the theoretical part should come last: we want people to get their hands on running code as fast as possible.